packages/wonder-stuff-server-google/src/__tests__/setup-integrations.test.js

```diff
@@ -0,0 +1,85 @@
+// @flow
```

Comment:
Copied from render-gateway `setupStackdriver` tests and modified to rename and update flow, etc.